bundles/org.corpus_tools.hexatomic.corpusedit/src/main/java/org/eclipse/wb/swt/SwtResourceManager.java

```diff
@@ -0,0 +1,454 @@
+/*******************************************************************************
+ * Copyright (c) 2011 Google, Inc. All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0 which accompanies this
+ * distribution, and is available at http://www.eclipse.org/legal/epl-v10.html
+ * Contributors: Google, Inc. - initial API and implementation
```

Comment:
```suggestion
 * 
 * <p>
 * Contributors: 
 * Google, Inc. - initial API and implementation
```

Improve readability while keeping the style valid.